README.md

```diff
@@ -35,131 +35,54 @@
 # After
 
 + BotD is a browser library for JavaScript bot detection
-+ Easily add ability to detect automation tools, browser spoofing and virtual machines
-+ Requires adding only 3 lines of JavaScript on your website
++ Easily add ability to detect automation tools
++ Requires adding only a few lines of JavaScript on your website
 ```
 
-### 🔩 [Try Demo](https://fingerprint.com/products/bot-detection/) - see the live demo running in your browser
+#### 🔩 [Try Demo](https://fingerprint.com/products/bot-detection/) - see the live demo running in your browser
 
-BotD runs in the [browser](#install-from-cdn); additionally you can harden it by using our open source cloud integrations.
+BotD runs in the [browser](#install); additionally you can harden it by using our open source cloud integrations.
 
-### Cloud Integrations ☁️
+## 🤖 **_Check for more advanced and accurate bot detection [here](https://fingerprint.com/products/bot-detection/)_**
 
-* [CloudFlare](https://github.com/fingerprintjs/botd-integrations/tree/main/cloudflare) - runs in CloudFlare workers for increased accuracy and security.
-* [Fastly](https://github.com/fingerprintjs/botd-integrations/tree/main/fastly/wasm) - runs in Fastly Compute@Edge high-performance WASM edge
-* [Next.js/Vercel](https://github.com/vercel/examples/tree/main/edge-functions/bot-protection-botd)<sup>3rd party</sup> - runs as a Next.js edge middleware
-
-## Install from CDN
-
-```html
-<script>
-    // Initialize an agent at application startup.
-    const botdPromise = import('https://openfpcdn.io/botd/v0.1')
-        .then( Botd => Botd.load({ publicKey: '<your-public-key>' }))
-    // Get the bot detection result when you need it.
-    // Result will contain the `requestId` property, that you can securely verify on the server.
-    botdPromise
-        .then(botd => botd.detect())
-        .then(result => console.log(result))
-        .catch(error => console.error(error))
-</script>
-```
-[Run this code](https://stackblitz.com/edit/botd-cdn?devtoolsheight=100&file=index.html)
-
-## Install from NPM to use with Webpack/Rollup/Browserify
+## Install
 
 ```bash
 npm i @fpjs-incubator/botd-agent
 # or
 yarn add @fpjs-incubator/botd-agent
 ```
 
+## Quick start
+
 ```js
-import Botd from '@fpjs-incubator/botd-agent';
+import { load } from '@fpjs-incubator/botd-agent'
 
 // Initialize an agent at application startup.
-const botdPromise = Botd.load({ publicKey: '<your-public-key>' });
-
-(async () => {
-  // Get the bot detection result when you need it.
-  // Result will contain the `requestId` property, that you can securely verify on the server.
-  const botd = await botdPromise;
-  const result = await botd.detect();
-  console.log(result);
-})();
+load()
+    // Get the bot detection result when you need it.
```

Comment:
These comments now make no sense because the code can't be split into different application parts like [shown here](https://dev.fingerprint.com/docs/js-agent-preloading).